board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
Another question, how to turn on vga console?
I've tried stderr/stdout serial,vga but it does not helps